tests/test_model.py

```diff
@@ -76,12 +75,9 @@ def test_cpes():
     assert ps2node
     ps3node = ProductNode.objects.create(parent=pv2node, obj=ps3)
     assert ps3node
+    ProductNode.objects.create(parent=ps1node, obj=pvariant1)
```

Comment:
I think we're just checking that node creation doesn't fail due to some unique constaint. It might not be needed but let's add that anyway for both variants.
```suggestion
    pvariant1node = ProductNode.objects.create(parent=ps1node, obj=pvariant1)
    assert pvariant1node
    pvariant2node = ProductNode.objects.create(parent=ps3node, obj=pvariant2)
    assert pvariant2node
```